Find a functionally equivalent code:android.content.RestrictionEntry.getDescription:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the provided user-visible description of the entry, if any.
 * @return the user-visible description, null if none was set earlier.
 */

@@ -1,3 +1,3 @@
 {
-    return description;
+    return mDescription;
 }

